nixos/modules/config/shells-environment.nix

```diff
@@ -107,7 +107,11 @@ in
     };
 
     environment.shellAliases = mkOption {
-      default = {};
```

Comment:
Are these defaults intentional?

Reply:
I just moved these aliases that were in `nixos/modules/programs/shell.nix` to here without any specific reason. Empty default is fine or better I think though.

Reply:
Never mind. It just reflected my preference.

Reply:
Reverted the default aliases. See https://github.com/NixOS/nixpkgs/pull/44441#discussion_r224580308